doc/source/georeferencing.md

```diff
@@ -0,0 +1,214 @@
+---
+file_format: mystnb
```

Comment:
Just a note. I have tried downloading the notebook from the doc page, using the download button at the top.
First it's not intuitive as it opens the notebook in JSON format in another browser tab. Then if I click on "downloading" on that page, I can get a .ipynb file, but when I open it with jupyter, I get an error.